build/BuildSteps.cs

```diff
@@ -115,7 +115,23 @@ public static void DotnetPublishForZips()
                 var outputPath = Path.Combine(Settings.OutputDir, runtime);
                 var rid = GetRuntimeId(runtime);
 
-                ExecuteDotnetPublish(outputPath, rid, "net8.0");
+                try
+                {
+                    // Attempt to publish for the current runtime
+                    ExecuteDotnetPublish(outputPath, rid, "net8.0");
+                }
+                catch (Exception ex)
+                {
+                    if (runtime.Contains("arm64"))
```

Comment:
Can we make this more specific to win-arm64? We're making changes to include linux-arm64 and I don't want that excluded (python is adding support in the next release)

Reply:
This approach did not work, so I will need to remove this. I started a thread on Teams. Jacob thinks we will need to update the Python worker itself to exclude it directly.